<launch>
  <arg name="output" default="screen"/>
  <arg name="camera" default="poll_cameras/cam2"/>
  <arg name="image" default="image_raw"/>

  <!-- apriltag detector node -->
  <node pkg="apriltag_ros" type="apriltag_detector_node"
    name="apriltag_detector" output="$(arg output)" ns="$(arg camera)">
    <param name="refine" type="bool" value="true"/>
    <param name="family" type="int" value="0"/> <!-- 0 = t36h11 -->
    <param name="type" type="int" value="0"/> <!-- MIT detector -->
    <param name="black_border" type="int" value="1"/>

    <remap from="~image" to="$(arg image)"/>
    <remap from="~apriltags" to="apriltags"/>
    <remap from="~image_detection" to="image_detection"/>
  </node>
</launch>
